dashboard/public/firebase-messaging-sw.js

```diff
@@ -0,0 +1,43 @@
+// Give the service worker access to Firebase Messaging.
+// Note that you can only use Firebase Messaging here. Other Firebase libraries
+// are not available in the service worker.
+importScripts(
+  "https://www.gstatic.com/firebasejs/9.2.0/firebase-app-compat.js"
+);
+importScripts(
+  "https://www.gstatic.com/firebasejs/9.2.0/firebase-messaging-compat.js"
+);
+
+// Initialize the Firebase app in the service worker by passing in
+// your app's Firebase config object.
+// https://firebase.google.com/docs/web/setup#config-object
+
+const firebaseConfig = {
+  apiKey: "AIzaSyCA4EFY7icetRO5ebk9IEZXTiVGb7Nd9NI",
```

Comment:
this values for security can be load from .env file
https://cli.vuejs.org/guide/mode-and-env.html#example-staging-mode